### PR TITLE
[Improvement] Support auto release note

### PR DESCRIPTION
**Pull request by grimoire**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Support auto generate release note.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by zhiqwang** [comment]
Great feature, does it automatically generate release reports based on each PR's labels?

**Comment by grimoire** [comment]
@zhiqwang It depends on how you define `report`. https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by irexyc — APPROVED** [review]

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        - Bug:P1
+        - Bug:P2
+        - Bug:P3
+    - title: 📚 Documentations
+      labels:
+        - documentation
+    - title: 🌐 Other
+      labels:
+        - '*'
+      exclude:
```
Comment (irexyc) [inline .github/release.yml:23]: Can we remove the exclude label?

Comment (grimoire) [reply .github/release.yml:23]: If we remove the exclude, PRs with any labels will be added to this group. include those Features and Bugs.

**Pull request merged**